src/sync-controller.js

```diff
@@ -516,7 +516,7 @@ export default class SyncController extends videojs.EventTarget {
         time: segment.start,
         accuracy: 0
       };
-    } else if (playlist.discontinuityStarts.length) {
+    } else if (playlist.discontinuityStarts && playlist.discontinuityStarts.length) {
```

Comment:
This will probably depend on how we plan to handle multi-period dash, and this extra check isn't problematic in anyway, but m3u8-parser always attaches an empty array for `discontinuityStarts`. If we plan to treat multi-period dash as discontinuities, then this check may be unnecessary, but still ok to be there

Reply:
Good call. I think we can just leave it in until we handle multi-period dash in the mpd-parser (so we know for sure).